### ISSUE TITLE
delete g.regeneratorRuntime throws in IE

### ISSUE THREAD

**Issue by AutoSponge** [facebook/regenerator#195]
I've tried to track this down but I can't find the source to patch.

I suggest changing this:

```js
var hadRuntime = g.regeneratorRuntime && Object.getOwnPropertyNames(g).indexOf("regeneratorRuntime") >= 0;
var oldRuntime = hadRuntime && g.regeneratorRuntime;
delete g.regeneratorRuntime;
```

To this:

```js
var hadRuntime = g.regeneratorRuntime && Object.getOwnPropertyNames(g).indexOf("regeneratorRuntime") >= 0;
var oldRuntime = hadRuntime && g.regeneratorRuntime;
try {
  delete g.regeneratorRuntime;
} catch (e) {
  g.regeneratorRuntime = undefined;
}
```

It doesn't fail silently now in IE but this will.

**Comment by sebmck** [comment]
Source is [runtime.js](https://github.com/facebook/regenerator/blob/master/runtime-module.js).

**Comment by benjamn** [comment]
Fixed by #196.
